brainscore_vision/model_helpers/activations/core.py

```diff
@@ -150,6 +248,23 @@ def _get_batch_activations(self, inputs, layer_names, batch_size):
         activations = self._unpad(activations, num_padding)
         return activations
 
+    def translate_images(self, images, shift):
+        assert type(images) == list
```

Comment:
```suggestion
        assert isinstance(images, list)
```
but even with that change, this is not very pythonic. Why does the type matter? Just do duck typing